bin/autoformat

```diff
@@ -2,7 +2,7 @@
 
 set -e
 
-rubocop -a -c rubocop.yml
+rubocop -A -c .rubocop.yml
```

Comment:
This has been broken for years and nobody noticed. Maybe we don't need this file at all?

Reply:
I'm fine with axing it